src/components/App.js

```diff
@@ -18,7 +20,7 @@ function App() {
         </Routes>
         <AddBook />
```

Comment:
- [ ] **[OPTIONAL]** This AddBook component appears in all routes. But it should only app in the root route (Book) for better code organization since the **Categories** component is somewhat dormant at this point. I would recommend you remove this component here and render it in the **Book** component instead.

Reply:
I have implemented this, thanks.

Reply:
Done- thanks